src/test/java/org/folio/impl/util/EventConfigHelperTest.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.impl.util;
+
+import org.apache.http.HttpStatus;
+import org.folio.cql2pgjson.exception.CQL2PgJSONException;
+import org.folio.rest.impl.util.EventConfigHelper;
+import org.folio.rest.persist.cql.CQLQueryValidationException;
+import org.junit.Test;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import static org.junit.Assert.*;
+
+public class EventConfigHelperTest {
+
+  @Test
+  public void mapToExceptionTest() {
+    Response response = EventConfigHelper.mapException(new CQL2PgJSONException("test"));
+    assertEquals(response.getStatus(), HttpStatus.SC_BAD_REQUEST);
+    assertEquals(response.getMediaType().toString(), MediaType.TEXT_PLAIN);
+
+    response = EventConfigHelper.mapException(new CQLQueryValidationException(null));
+    assertEquals(response.getStatus(), HttpStatus.SC_BAD_REQUEST);
+    assertEquals(response.getMediaType().toString(), MediaType.TEXT_PLAIN);
+
+    response = EventConfigHelper.mapException(new NullPointerException());
+    assertEquals(response.getStatus(), HttpStatus.SC_INTERNAL_SERVER_ERROR);
+    assertEquals(response.getMediaType().toString(), MediaType.TEXT_PLAIN);
+  }
+
+}
```

Comment:
There are two possibilities to assert for equality:
* `assertEquals(expected, actual)`
* `assertThat(actual, is(expected))` from org.hamcrest.MatcherAssert.assertThat

They differ in the order, the first one has `expected, actual`, the second `actual, expected`.

Please swap the parameters if you want to use `assertEquals`, or switch to `assertThat`.

Reply:
Fixed, thank you